src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="logo_box">
+        <a href="home" class="logo">
+          <img
+            src="images/logo.png"
+            alt="MOYO"
+            class="logo_img"
+          >
+        </a>
+      </div>
+      <nav>
+        <ul class="nav_ul">
+          <li class="nav_list">
+            <a href="#apple" class="is-active nav_link">
+            Apple
+            </a>
+          </li>
+          <li class="nav_list">
+            <a href="#apple" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_list">
+            <a href="#apple" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_list">
+            <a
+              href="#apple"
+              data-qa="hover"
+              class="nav_link"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_list">
+            <a href="#apple" class="nav_link">Gadgets</a>
```

Comment:
Why does the link lead to apple everywhere?
```suggestion
            <a href="#apple" class="nav_link">Gadgets</a>
```
```suggestion
            <a href="#gadgets" class="nav__ink">Gadgets</a>
```